Automattic/jetpack
# Add mocker runner for WAF blocked requests

#### Changes proposed in this Pull Request:

* Adds a new runner to the Mocker module in the Debug Helper that inserts mocked blocked requests into the WAF's database log.

#### Other information:


#### Jetpack product discussion


p1660581402530939-slack-C03LQF5FALB

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Ensure the Firewall is turned on in Jetpack Settings
* Ensure the **Jetpack Debug Tools** plugin is activated.
* In the Jetpack Debug menu, ensure the **Mocker** tool is activated.
* In the Jetpack Debug Mocker tool, select "Firewall blocked requests" from the `Data` dropdown, specify a number of entires to generate, and click "Run!"
* After completing, ensure that the `wp_jetpack_waf_blocklog` table now has new rows in it.
